src/components/pagination/pagination.stories.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import Pagination from ".";
+
+export default { title: "Pagination" };
```

Comment:
Please add the component name so the storybook docs addon can work effectively, see #41

```js
export default {
    title: 'Pagination',
    component: 'Pagination'
};
```